ros_alarms/alarms.py

```diff
@@ -113,13 +114,18 @@ def get_alarm(self):
 
         params = resp.alarm.parameters
         resp.alarm.parameters = params if params == '' else json.loads(params)
+        _last_alarm = resp.alarm
         return resp.alarm 
 
     def _severity_cb_check(self, severity):
+        # In case _last alarm hasnt been declared yet
+        if not '_last_alarm' in locals():
```

Comment:
This can be `self._last_alarm is not None` since it's declared as none in the init